app/src/main/java/com/infomaniak/drive/ui/addFiles/ImportFilesDialog.kt

```diff
@@ -73,7 +73,9 @@ class ImportFilesDialog : DialogFragment() {
         super.onDismiss(dialog)
         val errorCount = importCount - successCount
 
-        if (errorCount > 0) {
+        if (isLowMemory()) {
+            showSnackbar(R.string.uploadOutOfMemoryError, showAboveFab = true)
+        } else if (errorCount > 0) {
```

Comment:
If there's no importation error, the user will still get a out of memory snackbar regardless, which doesn't make much sens